packages/web-api/src/WebClient.ts

```diff
@@ -547,6 +548,28 @@ export class WebClient extends Methods {
         if (url.endsWith('admin.analytics.getFile')) {
           config.responseType = 'arraybuffer';
         }
+        // apps.event.authorizations.list will reject HTTP requests that send token in the body
```

Comment:
If this change does not break anything, we may want to apply this to all API methods. But if we make the change as small as possible, going with if condition this time sounds good to me too.

Reply:
I will add a comment here to note your point, but I would rather initially keep the scope of this change just to this one method; I am afraid that expanding to all methods may have unintended consequences.